### ISSUE TITLE
Shortcut to edit current directory

### ISSUE THREAD

**Issue by ghost** [fman-community/fman#859]
Please add shortcut which allows of editing current directory which is displayed on the top of the pane. Expected behavior is same as ctrl+l shortcut in Firefox or many other file managers.

**Comment by mherrmann** [comment]
Doesn't Ctrl+P let you do this?

**Comment by ghost** [comment]
> Doesn't Ctrl+P let you do this?

No, ctrl-p edit box does not contain current path. What I want to do is edit current path -- change some parts of it -- or copy parts of it.

**Comment by mherrmann** [comment]
I see. At the moment, you'd have to press Backspace to go up, then F11 to copy the path, then Ctrl+P, then Ctrl+V. You could write a plugin that simplifies this. See https://fman.io/docs/plugins-introduction.
